### PR TITLE
Feature/detailed warnings

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

**Comment by CloudNiner** [comment]
screen shot 2015-02-13 at 5 14 45 pm

File: src/nih_wayfinding/app/scripts/routing/directions-service.js [hunk 1]
```diff
@@ -167,6 +149,19 @@
             }
         }
 
+        function textForImage(imgUrl) {
```
Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/routing/directions-service.js:152]: I'm all ears on better text strings here...

File: src/nih_wayfinding/app/scripts/views/routing/directions/row-detail-directive.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+(function () {
+    'use strict';
+
+     /* ngInject */
+    function RowDetail($compile, Directions) {
+        // TODO: Animate the row appearing
```
Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/views/routing/directions/row-detail-directive.js:6]: @jfrankl I tried for a bit and failed :(

Pretty sure the ng-show/hide messes with things...insight welcome

**Comment by flibbertigibbet** [comment]
The term 'segment' might be a little to technical; how about 'section' or 'street' or some such?

+1

**Comment by jbranigan** [comment]
I agree with @flibbertigibbet . 
Pretty segment -> Pretty street
Segment is hazardous -> Block may have issues

**Comment by CloudNiner** [comment]
@jbranigan Done, via this PR and https://github.com/azavea/nih-wayfinding/commit/e775e87dfce071ecd041dba7e5ecbca58b3a979b

Thanks for checking in.

**Pull request closed without merging**